core/app/c/[communitySlug]/pubs/[pubId]/components/__tests__/author-pub-fixture.json

```diff
@@ -99,7 +97,6 @@
 				"children": [
```

Comment:
these fixtures don't need the `children` prop anymore!

Reply:
Nice catch!!!